File: problem_sets/week_1/cash/cash.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int centavos;
    do
    {
        centavos = get_int("Change owed: ");
    }
    while (centavos < 0);

    int moedas = 0;

    while (centavos / 25 > 0)
    {
        moedas += 1;
        centavos -= 25;
    }

    while (centavos / 10 > 0)
    {
        moedas += 1;
        centavos -= 10;
    }

    while (centavos / 5 > 0)
    {
        moedas += 1;
        centavos -= 5;
    }

    while (centavos / 1 > 0)
    {
        moedas += 1;
        centavos -= 1;
    }

    printf("%i\n", moedas);
}

File: problem_sets/week_6/sentimental_cash/cash.py
from cs50 import get_float

coins = 0
change = get_float("Change: ")

while change < 0:
    change = get_float("Change: ")

change *= 100

while change // 25 > 0:
    coins += 1
    change -= 25

while change // 10 > 0:
    coins += 1
    change -= 10

while change // 5 > 0:
    coins += 1
    change -= 5

while change // 1 > 0:
    coins += 1
    change -= 1

print(coins)
